### PR TITLE
Added example code for Go development against a traveler instance

### PR DESCRIPTION
**Pull request by raagaard**

### CODE REVIEW

File: traveler/examples/go/main.go [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+	outPretty.EmitUnpopulated = false
+
+	prettyOut, _ := outPretty.Marshal(reply)
+	fmt.Printf("Transfer successful, returned the following response: \n%s\n", prettyOut)
+}
+
+func parseOptions() (ctx context.Context, travelerEndpoint, credentialFile string, doLookup, doTransfer bool) {
+	travelerEndpoint = os.Getenv("TRAVELER_ENDPOINT")
+	if travelerEndpoint == "" {
+		travelerEndpoint = "grpc.a639386.traveler.stage.cipheruse.com:443"
```
Comment (bransorem) [inline traveler/examples/go/main.go:149]: Can we use an rVASP instead. Like `api.bob.vaspbot.net:443`

Comment (raagaard) [reply traveler/examples/go/main.go]: I used the same sample endpoint as the node example, I have tried switching it but it looks like there are now SSL cert issues.

I turned on skipping the SSL cert verification with a comment about how in production you should add the proper root CA's.

Comment (bransorem) [reply traveler/examples/go/main.go]: Yeah I don't think we should have used the stage ref in the first place. It may have predated the rVASPs

**Pull request merged**